src/main/java/Duke.java

```diff
@@ -1,8 +1,51 @@
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
 public class Duke {
-    public static void main(String[] args) {
-        String logo = "Hello! I'm Yube \n"
-                + "What can I do for you? \n" +
-                "Bye. Hope to see you again soon!";
-        System.out.println(logo);
+    private static final String LINE = "___________________________________________________________\n";
+
+    public static void main(String[] args) throws IOException {
+        BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
```

Comment:
Interesting how you used BufferedReader instead of Scanner. Is there any advantages of using it?